content/pt-br/kubernetes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Kubernetes
+status: Completed
+category: tecnologia
+tags: ["infraestrutura", "fundamental", ""]
+---
+
+## O que é
+
+Kubernetes, normalmente abreviado como K8s, é um orquestrador de contêineres de código aberto. 
+Ele automatiza o ciclo de vida de aplicações em containers em infraestruturas modernas, funciona como um "sistema operacional de datacenter" que gerencia aplicativos em um [sistema distribuído](/pt-br/distributed-systems/).
+
+O Kubernetes planeja os [contêineres](/pt-br/container/) entre [nós](/pt-br/nodes/) de um [cluster](/pt-br/cluster/), agrupando vários recursos de infraestrutura como balanceador de carga, armazenamento persistente, etc., para executar aplicações em contêineres.
+
+O Kubernetes permite automação e extensibilidade, permitindo que os usuários implantem aplicativos declarativamente (veja abaixo) de maneira reproduzível. É extensível por meio de sua [API](/pt-br/application-programming-interface/), permitindo que profissionais experientes do Kubernetes aproveitem seus recursos de automação de acordo com suas necessidades.
+
+## Problema relacionado
+
+A automação de infraestrutura e o gerenciamento declarativo de configuração são conceitos importantes há muito tempo, mas se tornaram mais prementes à medida que a [computação em nuvem](/pt-br/cloud-computing/) ganhou popularidade.
+À medida que a demanda por recursos de computação aumenta e as organizações precisam fornecer mais recursos operacionais com menos engenheiros, novas tecnologias e métodos de trabalho são necessários para atender a essa demanda.
+
+## Como isso ajuda
+
+De forma similar às ferramentas tradicionais de [infraestrutura como código](/pt-br/infrastructure-as-code/), o Kubernetes ajuda na automação e tem a vantagem de trabalhar com contêineres.
+Os contêineres são mais resistentes ao desvio de configuração do que [virtual](/pt-br/virtual-machine/) ou máquinas físicas.
```

Comment:
Minor: "do que máquinas virtuais ou físicas" or "do que máquinas físicas ou virtuais". Keep the virutal machine link.